app/models/ability.rb

```diff
@@ -40,7 +40,7 @@ def initialize(user)
       else
         school_scope = { id: user.school_id }
         related_school_scope = { school_id: user.school_id }
-        can :crud, Calendar, id: user.school.try(:calendar_id)
+        can [:update, :show], Calendar, id: user.school.try(:calendar_id)
```

Comment:
Can this be the same way round as the other one? Looks like there's a difference but there isn't.

Reply:
Yes, bad James J